### PR TITLE
[5716] New copy for waiting to be awarded teaching status

### PR DESCRIPTION
**Pull request by stevehook**
### Context
This PR implements a small copy change to make things clear for trainees that have been recommended for QTS, see https://becoming-a-teacher.design-history.education.gov.uk/register-trainee-teachers/making-it-clearer-that-a-trainee-is-waiting-to-be-awarded-teaching-status/

### Changes proposed in this pull request
Rather than leave the award date field blank we render a message to indicate when the trainee was recommended for qualification.

image

### Important business

- [x] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [x] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [x] Do we need to send any updates to DQT as part of the work in this PR?
- [x] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: app/components/award_details/view.rb [hunk 1]
```diff
@@ -11,7 +11,11 @@ def initialize(trainee)
     end
 
     def award_date
-      date_for_summary_view(trainee.awarded_at)
+      if trainee.awarded_at.present?
```
Comment (carlosmartinez) [inline app/components/award_details/view.rb:14]: can this be achieved with `trainee.awarded?`

**Review by carlosmartinez — APPROVED** [review]

File: app/components/award_details/view.rb [hunk 2]
```diff
@@ -11,7 +11,11 @@ def initialize(trainee)
...
+        date_for_summary_view(trainee.awarded_at)
+      elsif trainee.recommended_for_award?
+        "Waiting for award - met standards on #{date_for_summary_view(trainee.recommended_for_award_at)}"
```
Comment (darokel) [inline app/components/award_details/view.rb:17]: I remember us trying to put text in components into translations. I'm not sure how religiously that was followed but is it worth doing that here?

File: app/components/award_details/view.rb (hunk 1 above)
Comment (stevehook) [reply app/components/award_details/view.rb:14]: should be achievable that way but I just checked `select count(*) from trainees where state = 6 and awarded_at is null;` on prod and it returns 33 so it's (somehow) possible to have `awarded` state without an `awarded_at` value.

File: app/components/award_details/view.rb (hunk 2 above)
Comment (stevehook) [reply app/components/award_details/view.rb]: Done

**Review by defong — APPROVED** [review]

**Pull request merged**